internals/secrethub/service_aws_init.go

```diff
@@ -56,19 +56,55 @@ func (cmd *ServiceAWSInitCommand) Run() error {
 		cfg = cfg.WithRegion(cmd.region)
 	}
 
+	if cmd.role == "" {
+		role, err := ui.Ask(cmd.io, "What role would you like to use?")
+		if err != nil {
+			return err
+		}
+		cmd.role = role
+	}
+
+	if cmd.kmsKeyID == "" {
+		kmsKey, err := ui.Ask(cmd.io, "What KMS Key would you like to use?")
```

Comment:
I could imagine using something as: `What is the ID of the KMS-key you want to use for encrypting this service's credential? The service's IAM role should have decryption permissions on this key`. Or is this too extensive?

Reply:
Is `ui.AskAndValidate` nice to use here? Same for the role.